view/adminhtml/web/js/suggested_addresses_admin.js

```diff
@@ -43,20 +43,46 @@ define([
 
         initialize: function (config) {
             this._super();
-
             var self = this;
+            this.addressModal = this.buildModal();
+
+            if (config.controller === 'order_create') {
+                this.appendButtonToOrder();
+
+                if ('MutationObserver' in window) {
+                    var orderObserver = new MutationObserver(function (mutations) {
+                        var addressObserver = new MutationObserver(function (mutations) {
+                            self.appendButtonToOrder();
+                            orderObserver.disconnect();
+                        });
+
+                        self.appendButtonToOrder();
+
+                        if ($('.order-billing-address, .order-shipping-address').length) {
+                            addressObserver.observe($('.order-billing-address').get(0), {childList: true});
+                            addressObserver.observe($('.order-shipping-address').get(0), {childList: true});
+                        }
```

Comment:
Could this be split out into 2 separate `if` statements? Or is that `observe` method safe to call on a non-existent element?

Reply:
I tested this out and you were correct that it should have been split into two if statements. It has now been fixed.